pkg/cli/schemaherokubectlcli/apply.go

```diff
@@ -115,7 +126,14 @@ func ApplyCmd() *cobra.Command {
 	}
 
 	cmd.Flags().String("driver", "", "name of the database driver to use")
+
 	cmd.Flags().String("uri", "", "connection string uri to use")
+
+	cmd.Flags().String("username", "", "username to use when connecting")
+	cmd.Flags().String("password", "", "password to use when connecting")
+	cmd.Flags().StringSlice("host", []string{}, "hostname to use when connecting")
```

Comment:
should this be `hosts` instead?

Reply:
I don't think so...  Right now it will be:

`kubectl schemahero apply --host 10.1.1.1 --host 10.1.1.2`  I think hosts would be weird, IMO